src/imageproxy.rs

```diff
@@ -192,6 +193,42 @@ impl TryFrom<ImageProxyConfig> for Command {
     }
 }
 
+
+///Trying
+#[derive(Debug, serde::Deserialize)]
+pub struct LayersInfo {
+    ///Placeholder
+    pub Success: String,
+    ///Layers
+    pub Layers: Vec<BlobInfo>
+}
+
+///Algo
+#[derive(Debug, serde::Deserialize)]
+pub struct Algorithm {
+    name: String,
+    mime: String, 
+    prefix: Vec<u8>,
+    decompressor: String,
+    compressor:  String
+}
+
+///BlobInfo
+#[derive(Debug, serde::Deserialize)]
+pub struct BlobInfo {
+    ///Digest
+    pub Digest: String,
```

Comment:
Let's keep the Rust names as snake_case, and use serde annotations to describe what we're expecting.  See e.g. https://serde.rs/field-attrs.html#rename

In this case e.g.
```suggestion
    #[serde(rename = "Digest")] 
    pub digest: String,
```

We might be able to use `rename-all = "PascalCase"` on the toplevel, although "URLs" may be an exception?